server/modules/taxonomy/repository/taxonomyRepository.js

```diff
@@ -412,5 +417,5 @@ export const deleteDraftTaxaByTaxonomyUuid = async (surveyId, taxonomyUuid, clie
 
 const toSearchValue = (filterValue) =>
   filterValue
-    ? R.pipe(R.ifElse(R.is(String), R.identity, R.toString), R.trim, R.toLower, R.replace(/\*/g, '%'))(filterValue)
+    ? A.pipe(R.ifElse(R.is(String), R.identity, R.toString), R.trim, R.toLower, R.replace(/\*/g, '%'))(filterValue)
```

Comment:
this can be simplified with`String(filterValue).trim().toLowerCase().replace(/\*/g, '%')`